albumentations/augmentations/geometric/transforms.py

```diff
@@ -251,6 +252,28 @@ def apply_to_bbox(self, bbox, random_state=None, **params):
         bbox_returned = bbox_from_mask(mask)
         bbox_returned = F.normalize_bbox(bbox_returned, rows, cols)
         return bbox_returned
+    
+    def apply_to_keypoint(self, keypoint: KeypointInternalType, random_state=None, half_win=9, **params) -> KeypointInternalType:
+        """
+            Only consider 2-D case.
+        """
+        image = np.zeros([params["cols"], params["rows"]])
+        X, Y = np.meshgrid(np.arange(image.shape[0]), np.arange(image.shape[1]))
+        image[Y, X] = np.exp(- 0.5 / (0.02 ** 2) *
+                             (((X - keypoint[0]) / image.shape[0]) ** 2 + ((Y - keypoint[1]) / image.shape[1]) ** 2))
+        remap_image = F.elastic_transform(
+            image,
+            self.alpha,
+            self.sigma,
+            self.alpha_affine,
+            self.interpolation,
+            self.border_mode,
+            self.mask_value,
+            np.random.RandomState(random_state),
+            self.approximate,
+        )
+        interp_y, interp_x = np.where(remap_image == np.max(remap_image))
+        return (interp_x[0], interp_y[0], 0.0, 0.0)
```

Comment:
What if the keypoint outside of the image? Looks like we always will find keypoint inside the image and in the case  when we can not see the keypoint

Reply:
Yes, there is such a problem, but I can't think of a simple and elegant way to solve it. Therefore, you should try to ensure that the keypoints do not go beyond the scope of the image, otherwise it will return incorrect results.